src/libcore/option.rs

```diff
@@ -10,9 +10,9 @@
 
 //! Optional values
 //!
-//! Type `Option` represents an optional value: every `Option`
+//! Type `Option<T>` represents an optional value: every `Option<T>`
```

Comment:
As I've just entered the Rust community I don't think I have (and don't intend to have) a strong vote here.

@apasel422, @steveklabnik, please let me just know what you both agree on and I am more than happy to adjust the RFC whatever you come up with :)